api-reference/docstubs/api/activitystatistics-list.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "List activityStatisticses"
+description: "List properties and relationships of the activityStatistics objects."
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# List activityStatisticses
```

Comment:
Line 10: "activityStatisticses"?
Did your code append "es" at the end and form a new word?

Reply:
Yup - dupe